modules/ocf_desktop/manifests/xfce.pp

```diff
@@ -3,7 +3,6 @@
     recommends => false;
   }
 
-  # TODO: figure out why this gets installed anyway on new systems
   # xfce4-power-manager asks for admin authentication on login in order to
   # "change laptop display brightness"
   package { 'xfce4-power-manager':
```

Comment:
Yeah, that's true, my concern was just that this hadn't run on all the desktops, but that doesn't really make sense, since all the desktops have run puppet pretty recently.